.github/workflows/migration.yml

```diff
@@ -0,0 +1,24 @@
+name: Migration
+
+on:
+  workflow_dispatch:
+    inputs:
+      block:
+        description: 'Block number'    
+        required: true
```

Comment:
Adds a new manual job that requires the block number as input -> https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/